ghost/core/core/server/services/comments/CommentsService.js

```diff
@@ -216,8 +216,9 @@ class CommentsService {
      * @param {string} member - The ID of the Member to comment as
      * @param {string} comment - The HTML content of the Comment
      * @param {any} options
+     * @param {Date} createdAt - Optional custom created_at timestamp
```

Comment:
```suggestion
     * @param {Date} [createdAt] - Optional custom created_at timestamp
```

Adding the brackets will tell JSDoc that this parameter is optional, else we'll get linter errors if we try to call this without createdAt